server/routes/visitJourney/overbooking.ts

```diff
@@ -0,0 +1,41 @@
+import type { Request, Response } from 'express'
+import getUrlPrefix from './visitJourneyUtils'
+import { VisitSessionsService } from '../../services'
+
+export default class Overbooking {
+  constructor(
+    private readonly mode: string,
+    private readonly visitSessionsService: VisitSessionsService,
+  ) {}
+
+  async viewFromConfirm(req: Request, res: Response): Promise<void> {
```

Comment:
```suggestion
  async get(req: Request, res: Response): Promise<void> {
```
Doesn't really matter, but this matches the naming convention in this part of the app (which I know is different to newer sections / other apps 🙃!)

Reply:
I'll leave it as is for now - as I believe my way of implementing the date and time post will have us requiring a different function within this controller - but will wait and see